src/index.html

```diff
@@ -62,7 +62,7 @@ <h1 class="content__title">Web development project</h1>
         <button class="arrow-down__button"></button>
       </div>
 
-      <img src="./images/first_bg.jpeg" class="first__background">
+      <img src="./images/first_bg.jpeg" class="first__background" alt="First section background">
```

Comment:
It's better to describe what you see in the image, because alt tags mostly need for blind people, and also they can affect on SEO, so key words are encouraged.